docs/psa-transition.md

```diff
@@ -743,6 +743,10 @@ A curve is fully determined by a curve family identifier and the private key siz
 | `MBEDTLS_ECP_DP_SECP256K1` | [`PSA_ECC_FAMILY_SECP_K1`](https://mbed-tls.readthedocs.io/projects/api/en/development/api/group/group__crypto__types/#group__crypto__types_1ga48bb340b5544ba617b0f5b89542665a7) | 256 |
 | `MBEDTLS_ECP_DP_CURVE448` | [`PSA_ECC_FAMILY_MONTGOMERY`](https://mbed-tls.readthedocs.io/projects/api/en/development/api/group/group__crypto__types/#group__crypto__types_1ga1f624c5cdaf25b21287af33024e1aff8) | 448 |
 
+The following helper functions can be used to convert between the 2 types:
```

Comment:
Most readers should use these conversion functions and won't need the table. So please move this text before the table. Something like:

> 
A curve is fully determined by a curve family identifier and the private key size in bits. You can use the following functions to convert between the PSA and legacy elliptic curve designations: (…)
>
> The following table gives the correspondence between legacy and PSA elliptic curve designations. (…)